rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -158,13 +161,41 @@ def _update_ens_mappings_schema(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _update_ens_mappings_schema')
 
 
+def _fix_kraken_eth2_events(write_cursor: 'DBCursor') -> None:
+    """Fix kraken events with negative amounts related to staking ETH after the merge"""
+    log.debug('Enter _fix_kraken_eth2_events')
+    write_cursor.execute(
+        'SELECT identifier, amount, usd_value FROM history_events WHERE location="B" AND '
+        'asset=? AND type="staking" AND subtype="reward" AND CAST(amount AS REAL) < 0',
+        (A_ETH2.identifier,),
+    )
+    update_tuples = []
+    for event_row in write_cursor:
```

Comment:
Not sure what's better, but this can also be done in a single sql query I guess

Reply:
The problematic point can be I guess in checking whether `amount` is less than zero since `REAL` in sqlite is not super-precise, but this can be checked with sqlites's method like amount.startswith('-')

Reply:
It will help in case there will be many kraken events to modify, but if you think my proposal is too ugly, let's keep the code as it is right now

Reply:
I did went with something similar but the problem is that we have to change the sign in two column and this I believe can be an issue since I'm not sure how good it is and if we will loose pricesion. That is why I opted for this approach instead